app/src/ui/octicons/donut.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react'
```

Comment:
Only a nit about this file: should it live in `app/src/ui` instead of `app/src/ui/octicons`?

Reply:
:D I kept going back and forth about that.... because I was thinking if I saw that on the screen, I would think of it like our other icons.. and go looking around the octicons.. but.. also agree it isn't one. :P Since, it felt weird there to you I will gladly move it. :)